bitcoin.md

```diff
@@ -52,13 +52,13 @@ $ sudo make install
 ```
 $ sudo bash -c 'cat <<EOF> /etc/bitcoin/bitcoin.conf
 server=1
+daemon=1
 disablewallet=0
-datadir=/bitcoin
-rpcuser=btcuser
-rpcpassword=btcpass
-rpcallowip=0.0.0.0/0
-bind=0.0.0.0:8333
-rpcbind=0.0.0.0:8332
+rpcuser=1
+rpcpassword=1
+rpcallowip=127.0.0.1
+bind=127.0.0.1:8333
+rpcbind=127.0.0.1:8332
```

Comment:
if adjusting for loopback, update the `Note` above this section specifiying that the node will not be accessible outside of localhost.

Reply:
https://github.com/bitnet-io/stacks-core-bitnet/releases/tag/Stacks-2.0-mining-for-Bitcoin-people

Reply:
> if adjusting for loopback, update the `Note` above this section specifiying that the node will not be accessible outside of localhost.

for the regular common user IT SHOULD NOT BE OPEN TO THE WAN
even production system will not open their rpcallowip to the WAN for safety reasons

Reply:
> > if adjusting for loopback, update the `Note` above this section specifiying that the node will not be accessible outside of localhost.
> 
> for the regular common user IT SHOULD NOT BE OPEN TO THE WAN even production systems dealing with any sum of money in BTC will not open their rpcallowip to the WAN for safety reasons

90% of everyone using this or TRYING to use this is running stacks-node and bitcoind on the same local box

Reply:
> > > if adjusting for loopback, update the `Note` above this section specifiying that the node will not be accessible outside of localhost.
> > 
> > 
> > for the regular common user IT SHOULD NOT BE OPEN TO THE WAN even production systems dealing with any sum of money in BTC will not open their rpcallowip to the WAN for safety reasons
> 
> 90% of everyone using this or TRYING to use this is running stacks-node and bitcoind on the same local box

people arent launching stacks-node on a remote system to connect to their local box with bitcoind
most of the time someone is syncing bitcoind or bitcoin-qt locally on their machine and then simply running stacks-node also on their local machine to interface with bitcoind or bitcoin-qt